.github/workflows/release.yml

```diff
@@ -0,0 +1,210 @@
+name: Release
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+jobs:
+  update:
+    name: Update
+    runs-on: ubuntu-latest
+    outputs:
+      hydra_eval: ${{ steps.update.outputs.hydra_eval }}
+      nix_release: ${{ steps.update.outputs.nix_release }}
+      updated: ${{ steps.update.outputs.updated }}
+    steps:
+      - uses: cachix/install-nix-action@v13
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure git
+        run: |
+          git config user.name 'github-actions[bot]'
+          git config user.email '41898282+github-actions[bot]@users.noreply.github.com'
+
+      - name: Update from latest Hydra eval
+        id: update
+        run: |
+          hydra_eval="$(curl -sLH 'Accept: application/json' https://hydra.nixos.org/jobset/nix/master/latest-finished | jq '.jobsetevals[0]')"
```

Comment:
What do you think of skipping releases that don't have a 100% green status?

Reply:
I might be misremembering, but won't the `latest-finished` endpoint only return the latest green jobset?

Reply:
That seems to be the case: https://github.com/NixOS/hydra/blob/fa924ea6978f2d47536a418157984a881c8226fc/src/lib/Hydra/Base/Controller/ListBuilds.pm#L76

Reply:
Ah I realize I may not have paid close enough attention. That endpoint will redirect to the latest successful job in any evaluation of the jobset, and given there isn't one job with constituents to show success/fail status of all of the relevant builds, this won't guarantee that one wouldn't have failed, will it?

Do you have any suggestions to find only the latest 100% successful eval?

Given the fewer details you get from Hydra via the JSON responses (e.g. you don't get the number of successful/failed jobs from the evaluation list), I'm not sure how better to handle that other than querying all of the jobs in the latest eval for failures.